extensions/ql-vscode/package.json

```
@@ -503,7 +503,13 @@
       },
       {
         "command": "codeQLQueries.runLocalQueryFromQueriesPanel",
-        "title": "Run local query"
+        "title": "Run local query",
+        "icon": "$(run)"
+      },
+      {
+        "command": "codeQLQueries.runLocalQueriesFromPanel",
```

```suggestion
        "command": "codeQLQueries.runLocalQueriesFromQueriesPanel",
```
Should we name this to match https://github.com/github/vscode-codeql/blob/6adb7722d306fb10cbaa40027ab86f25f5989b8b/extensions/ql-vscode/package.json#L505

---

yes, I think in the end I would match Shatis style and remove the Queries Panel words from this since the prefix already specifies that..

---

I can do it here for both?

---

Using the `codeQLQueries.` prefix to mean it's for the queries panel works for me.

I'd prefer to not rename the rest of the existing commands in this PR. If we do that in a separate PR it would avoid holding up this PR and make the history more understandable.

---

So, rename the command being added in this PR to `codeQLQueries.runLocalQueries`?
Do you want to do that in this PR, or merge this as it is and rename all three in a followup?

---

Lets merge this as is and rename in a followup!